devlib/target.py

```diff
@@ -1749,19 +1755,39 @@ def __init__(self, text):
                 name, value = line.split('=', 1)
                 self._config[name.strip()] = value.strip()
 
-    def get(self, name, strict=False):
+    def __iter__(self):
+        return iter(self._config)
+
+    def __len__(self):
+        return len(self._config)
+
+    def iteritems(self):
+        return iter(self.items())
+
+    def __getitem__(self, name):
         name = self.get_config_name(name)
-        res = self._config.get(name)
+        try:
+            return self._config[name]
+        except KeyError:
+            raise KernelConfigKeyError(
+                "{} is not exposed in kernel config".format(name),
+                name
```

Comment:
Not really necessary, but it can be handy in exception handling code:
```
try:
    kernel_config.do_something_that_raises('FOO')
except KernelConfigTristate as e:
   config_opt = e.args[1]
   raise AnotherExceptionWithMoreContext('hello world: {}'.format(config_opt))
```